src/components/common/ProfilePanel.js

```diff
@@ -166,45 +183,42 @@ class ProfilePanel extends React.Component {
     );
   };
 
-  renderName = () => {
-    if (!this.state.editingName) {
+  const renderName = () => {
+    if (!editingName) {
       return (
         <div
           className="panel-name"
-          onMouseEnter={() => this.setState({ nameIsHovering: true })}
-          onMouseLeave={() => this.setState({ nameIsHovering: false })}
-          onClick={this.handleEditNameClick}
+          onMouseEnter={() => setNameHover(true)}
+          onMouseLeave={() => setNameHover(false)}
+          onClick={handleEditNameClick}
         >
-          <div className="panel-name-text">{this.props.displayName || 'Joe Bruin'}</div>
-          {this.state.nameIsHovering && (
-            <button className="edit-icon-image" onClick={this.handleEditNameClick}>
+          <div className="panel-name-text">{name || 'Joe Bruin'}</div>
```

Comment:
Is this change intentional?
```suggestion
          <div className="panel-name-text">{displayName || 'Joe Bruin'}</div>
```

Reply:
Yeah, name is the state while displayName is the prop. The name changing test failed when I used displayName rather than name. As far as I can tell, using either variable works as intended.

Reply:
Sounds good to me, this would be good to also test manually to make sure that unintended side effects happen in this change <since we also changed the tests itself>

(I already did but just noting for you)